x/budget/spec/03_state_transitions.md

```diff
@@ -2,16 +2,20 @@
 
 # State Transitions
 
-This document describes the state transaction operations pertaining to the budget module.
+The state transaction operations for the budget module are:
 
 - CollectBudgets
+
 - TotalCollectedCoins
+
 ## CollectBudgets
 
 Get all budgets registered in `params.Budgets` and select the valid budgets to collect budgets for the block by its respective plan.
-This state transition occurs at each `BeginBlock`. You can read more about it in [04_begin_block.md](04_begin_block.md).
+
+This state transition occurs at each `BeginBlock`. See [Begin-Block](04_begin_block.md).
 
 ## TotalCollectedCoins
 
 `TotalCollectedCoins` are accumulated coins in a budget since the creation of the budget.
-This state transition occurs at each `BeginBlock` with `CollectBudgets`. You can read more about it in [04_begin_block.md](04_begin_block.md).
+
+This state transition occurs at each `BeginBlock` at the same time as the `CollectBudgets` operation. See [Begin-Block](04_begin_block.md).
```

Comment:
do both of these operations occur at the same time? (I wasn't sure what we meant by "with")

Reply:
`with` refers to when `CollectedBudgets` operation is triggered at each block, `TotalCollectedCoins` operation is also triggered. For now, let's just take out `at the same time as the CollectBudgets operation` part.
```suggestion
This state transition occurs at each `BeginBlock`. See [Begin-Block](04_begin_block.md).
```